docs/ref/urlresolvers.txt

```diff
@@ -30,6 +30,9 @@ you can use any of the following to reverse the URL::
 
     reverse(views.archive)
 
+The callable view object can also be a function view :ref:`derived from a
+class-based view <view-function-from-cbv>`.
+
```

Comment:
I think instead of adding a section to class-based-views and a reference here, I think we should add a note here instead. 
Passing a callable is actively discouraged as you can't reverse namespaced views (see ticket-17914). I don't think we should add a section to another part of the docs motivated by making this work for CBVs which doesn't include the context where it is discouraged :thinking: 

I'm not 100% sure this is worth documenting at all, what do you think?

Reply:
Thank you @sarahboyce for the review!

I'd added the section mainly because I couldn't find a way to add the how-to/example part in a note -- necessary to point out that you shouldn't use the result of `as_view` directly. I removed the section and incorporated the concern in the note in `urlresolvers.txt`. LMKWYT

As to whether this is worth documenting, I'm not sure by what's the criteria we use to determine whether something is worth documenting. I'm thinking, sure reversing by instance isn't the best way Django reverses views, but should it be documented? I don't know... but since this ticket is open, I'm thinking, let's add a note.

Reply:
I hadn't noticed that ``urlconf`` arg is documented at the bottom, but I agree that it should have been somewhere above the notes beginning with ``If no match can be made, reverse() raises a NoReverseMatch.``

I'm okay with having an admonition at the bottom. Looks better that way, thanks.